diff -Naur udisks-2.1.3.ORIG/src/tests/test.c udisks-2.1.3/src/tests/test.c
--- udisks-2.1.3.ORIG/src/tests/test.c	2014-03-05 17:05:43.000000000 +0100
+++ udisks-2.1.3/src/tests/test.c	2014-06-24 20:19:29.580988065 +0200
@@ -299,14 +299,6 @@
   UDisksSpawnedJob *job;
   gchar *s;
 
-  s = g_strdup_printf (UDISKS_TEST_DIR "/udisks-test-helper 4");
-  job = udisks_spawned_job_new (s, NULL, getuid (), geteuid (), NULL, NULL);
-  _g_assert_signal_received (job, "completed", G_CALLBACK (on_completed_expect_failure),
-                             (gpointer) "Command-line `./udisks-test-helper 4' was signaled with signal SIGSEGV (11): "
-                             "OK, deliberately causing a segfault\n");
-  g_object_unref (job);
-  g_free (s);
-
   s = g_strdup_printf (UDISKS_TEST_DIR "/udisks-test-helper 5");
   job = udisks_spawned_job_new (s, NULL, getuid (), geteuid (), NULL, NULL);
   _g_assert_signal_received (job, "completed", G_CALLBACK (on_completed_expect_failure),

