diff -Naur libtool-2.4.2.ORIG/tests/runpath-in-lalib.at libtool-2.4.2/tests/runpath-in-lalib.at
--- libtool-2.4.2.ORIG/tests/runpath-in-lalib.at	2011-10-09 23:45:20.000000000 +0200
+++ libtool-2.4.2/tests/runpath-in-lalib.at	2014-06-26 22:11:48.661793143 +0200
@@ -25,6 +25,11 @@
 AT_SETUP([Runpath in libtool library files])
 AT_KEYWORDS([libtool])
 
+# if hardcode_into_libs=yes libtool explicitly does NOT
+# put rpath in the .la file
+eval "`$LIBTOOL --config | $EGREP '^hardcode_into_libs='`"
+AT_CHECK([test x"${hardcode_into_libs:-}" = x"yes" && exit 77])
+
 AT_DATA([a.c],
 [[int a () { return 0; }
 ]])
